### DISCUSSION TITLE
Error: failed to rename container named xxx-xxx-job-7793c6da151f5fd44c99ffad5f44ebdbaa7eaebe

### DISCUSSION THREAD

**Discussion by RayNawara** [basecamp/kamal#373] [General]
Hi! I get this error a lot after my initial setup. I can get by it if I rebuild my server and rerun setup.

Just wondering if anyone knows how I can fix it.

`Finished all in 132.6 seconds
  ERROR (SSHKit::Command::Failed): Exception while executing on host x.x.x.x: docker exit status: 1
docker stdout: Nothing written
docker stderr: Error response from daemon: sandbox 0c02be859d855b53c6f67bd3193c9d99fa31ad3b1dbd90617d75f496807b4f07 not found
Error: failed to rename container named xxx-xxx-job-7793c6da151f5fd44c99ffad5f44ebdbaa7eaebe`

**Comment by AliOsm** [comment]
I'm facing the same issue currently with cron jobs container.

**Comment by kyrylo** [comment]
I stumbled upon this error as well.

You can see this error when you deploy a fresh container that keeps failing.

Just fix your container so that it doesn't restart all the time, and then the subsequent deploys will work.

While you are fixing it, you will have to `kamal app remove` before you execute `kamal deploy`.
